test/linter/test_swift_appleswiftformat.vader

```diff
@@ -0,0 +1,37 @@
+Before:
+  call ale#assert#SetUpLinterTest('swift', 'appleswiftformat')
+
+After:
+  call ale#assert#TearDownLinterTest()
```

Comment:
Call  ale#test#RestoreDirectory() if using ale#test#SetDirectory() to reset the g:dir variable.